hostedzones/service.justice.gov.uk.yaml

```diff
@@ -103,10 +102,6 @@ _d8e015eb77621a22d1bb4fe339ec1ebd.dev.manage-external-funded-offender-provision:
   ttl: 300
   type: CNAME
   value: _f45d3bd1f2f3aa9f672facf92f66d56a.sdgjtdhdhz.acm-validations.aws.
-_d0056e0292b63e43ee7490de450c6111.exchange-integration-services-stream:
```

Comment:
Please can you exclude this one for now as we are still awaiting the cert to be generated.